src/components/AddItems.js

```diff
@@ -9,9 +9,13 @@ const arrayUnion = firebase.firestore.FieldValue.arrayUnion;
 const AddItemsToList = () => {
   const [shoppingListItemName, setShoppingListItemName] = useState('');
   const [daysLeftForNextPurchase, setDaysLeftForNextPurchase] = useState(7);
+  const [shoppingListItemNameExists, setShoppingListItemNameExists] = useState(
+    false,
+  );
```

Comment:
Great work! well done guys. A quick observation. I can see that the initial value of shoppingListItemNameExists here is false.